src/components/ArticleSquare.astro

```diff
@@ -0,0 +1,23 @@
+---
+const { entry } = Astro.props;
+const _date_obj = new Date(entry.data.date);
+const date = {
+	month: _date_obj.getMonth() + 1,
+	day: _date_obj.getDate(),
+	year: _date_obj.getFullYear(),
+};
+const url = `/opinion/${entry.id}`; // todo: find a way to use this
+// and to have one for author too.
+---
+
+<div
```

Comment:
seems indents are messed up here, we will need to clean it up and turn it into 4 (or 2!) spaces instead of tabs, since the indent level is comical here.

Reply:
we'll work with 2 spaces!